spirv_cross.hpp

```diff
@@ -1145,6 +1145,8 @@ class Compiler
 	bool has_extended_member_decoration(uint32_t type, uint32_t index, ExtendedDecorations decoration) const;
 	void unset_extended_member_decoration(uint32_t type, uint32_t index, ExtendedDecorations decoration);
 
+	bool check_internal_recursion(SPIRType &type, std::unordered_set<uint32_t> &checked_ids);
```

Comment:
const SPIRType

Reply:
Fixed. And in `type_contains_recursion()`.